src/modules/main/store/mainSlice.ts

```diff
@@ -58,8 +59,9 @@ export default mainSlice.reducer;
 
 export const fetchImportantPosts = (): AppThunkType => async (dispatch) => {
   try {
-    const posts = await Promise.resolve(MOCK_POSTS);
-    dispatch(loadImportant(posts));
+    const posts = await Promise.resolve(getPosts('important'));
```

Comment:
`getPosts` already returns a promise
```suggestion
    const posts = await getPosts('important');
```

Reply:
Thank you 💪🏻